src/routes/safe/components/FetchError.tsx

```diff
@@ -25,14 +26,14 @@ const ErrorContainer = styled.div`
   margin-top: -30px;
 `
 
-const FetchError = ({ text, buttonText, redirectRoute }: FetchErrorProps): ReactElement => {
+const FetchError = ({ text, buttonText, redirectRoute, onClick }: FetchErrorProps): ReactElement => {
   return (
     <ErrorContainer>
       <img src="./resources/error.png" alt="Error" />
 
       <Title size="xs">{text}</Title>
 
-      <StyledLink to={redirectRoute}>
+      <StyledLink to={redirectRoute} onClick={() => onClick?.()}>
```

Comment:
Can't we just pass onClick itself? If it's undefined, it won't be assigned. `onClick={onClick}`